uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -378,6 +378,29 @@ _Response Fields_
   Added in UAA 3.3.0
 </aside>
 
+When an access token is revoked using this endpoint, the UAA Introspect Token endpoint (``/introspect``)
+will respond with ``"active": false`` when presented with the revoked access token.
+
+If the access token is in the JWT format (as opposed to the opaque format), the server config ``uaa.jwt.revocable`` or
+the Identity Zone config ``config.tokenPolicy.jwtRevocable`` must be set to ``true`` for
+this feature to work. However, OAuth resource servers are generally not required to call the UAA Introspect
```

Comment:
> this feature to work

maybe "for the revocation to work"?

I may also be useful to document what the UAA returns if these settings are `false` and someone tries to revoke a token?

Reply:
Point discussed and/or edits made.